charts/sourcegraph/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+{{- "\n" }}
+
+Thank you for installing {{ .Chart.Name }}.
+
+Your release is named {{ .Release.Name }}.
+
+To learn more about supported configuration, check out https://docs.sourcegraph.com/admin/install/kubernetes/helm.
+
+{{- "\n" }}
+
+{{- if not .Values.prometheus.privileged }}
+🚧 Warning 🚧
+
+You have set 'prometheus.privileged' to 'false'. As a result, our built-in observability stack will not function properly. 
```

Comment:
We should include more details here about what won't work so that customers can make an informed choice.